lib/content-services/src/lib/content-node-selector/content-node-selector.component.spec.ts

```diff
@@ -393,4 +394,28 @@ describe('ContentNodeSelectorComponent', () => {
             expect(tabGroup).toBe(undefined);
         });
     });
+
+    describe('Selected nodes counter', () => {
+        it('should getSelectedCount return 0 by default', () => {
+            expect(component.getSelectedCount()).toBe(0);
+        });
+
+        it('should getSelectedCount return 1 when a node is selected', () => {
+            component.onSelect([new Node({ id: 'fake' })]);
+
+            expect(component.getSelectedCount()).toBe(1);
+        });
+
+        it('should have injected the counter text', () => {
+            fixture.detectChanges();
+            fixture.whenStable().then(() => {
+                const breadcrumb = fixture.debugElement.query(By.directive(DropdownBreadcrumbComponent));
+                expect(breadcrumb).not.toBeNull('Breadcrumb not found');
```

Comment:
Why do we need to check the breadcrumb?

Reply:
Not sure why I left that in. I think I assumed the breadcrumb would have to be present for the counter to also be present, when that's not true. Removing.